api-contract/src/main/java/uk/gov/hmcts/fees2/register/api/contract/request/FeeDto.java

```diff
@@ -47,4 +47,7 @@
 
     protected String keyword;
 
+    @JsonProperty("reason_for_update")
+    protected String reasonForUpdate;
```

Comment:
What is the requirement to add 'reasonForUpdate' in as attribute in FeeDTO Model ?

Reply:
This also updated but forgot to push, did that along with Encode